src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ChangeRequestsTab.tsx

```diff
@@ -0,0 +1,52 @@
+import { ChangeRequest, WorkPackage } from 'shared';
+import { useAllChangeRequests } from '../../../hooks/change-requests.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { projectWbsPipe } from '../../../utils/pipes';
+import ChangeRequestRow from '../../../components/ChangeRequestRow';
+interface ChangeRequestsTabProps {
+  workPackage: WorkPackage;
+}
+
+const ChangeRequestsTab: React.FC<ChangeRequestsTabProps> = ({ workPackage }) => {
+  const { data: changeRequests, isError: crIsError, isLoading: crIsLoading, error: crError } = useAllChangeRequests();
+  const currentDate = new Date();
+
+  if (crIsLoading || !changeRequests) return <LoadingIndicator />;
+  if (crIsError) return <ErrorPage message={crError?.message} />;
+
+  const crUnreviewed = changeRequests
+    ? changeRequests
+        .filter((cr: ChangeRequest) => !cr.dateReviewed && projectWbsPipe(cr.wbsNum) === projectWbsPipe(workPackage.wbsNum))
+        .sort((a, b) => b.dateSubmitted.getTime() - a.dateSubmitted.getTime())
+    : [];
+
+  const crApproved = changeRequests
+    ? changeRequests
+        .filter(
+          (cr: ChangeRequest) =>
+            cr.dateReviewed &&
+            cr.accepted &&
```

Comment:
if a cr is accepted, it is implicitly reviewed so you should only need to check cr.accepted, not also cr.dateReviewed